src/resources/views/crud/components/datagrid.blade.php

```diff
@@ -0,0 +1,32 @@
+<div class="datagrid p-3">
+    @foreach($columns as $column)
+        <div class="datagrid-item">
+            <div class="datagrid-title">{!! $column['label'] !!}</div>
+            <div class="datagrid-content">
+                @php
+                // create a list of paths to column blade views
+                // including the configured view_namespaces
+                $columnPaths = array_map(function($item) use ($column) {
+                    return $item.'.'.$column['type'];
+                }, \Backpack\CRUD\ViewNamespaces::getFor('columns'));
+
+                // but always fall back to the stock 'text' column
+                // if a view doesn't exist
+                if (!in_array('crud::columns.text', $columnPaths)) {
+                    $columnPaths[] = 'crud::columns.text';
+                }
+                @endphp
```

Comment:
We can prooobably move this to the class file... but I didn't want to do that in the first try... wanted to keep the EXACT same code that we had in `show_table.blade.php`